net/Socket.cpp

```diff
@@ -365,13 +365,20 @@ int SocketPoll::poll(int64_t timeoutMaxMicroS)
     if (_pollFds[size].revents)
     {
         // Clear the data.
+        //Size of wakeup array
+        size_t arraySize = sizeof(_wakeup) / sizeof(_wakeup[0]);
 #if !MOBILEAPP
         int dump[32];
-        dump[0] = ::read(_wakeup[0], &dump, sizeof(dump));
-        LOG_TRC("Wakup pipe read " << dump[0] << " bytes");
+        //Empty wakeup pipe inside poll function.
+        for (size_t ii = 0; ii < arraySize; ii++)
```

Comment:
Here you can use "i" instead of "ii". Scope will be in loop so don't need to concern about it.